runtime-watcher/tests/e2e/e2e_test.go

```diff
@@ -0,0 +1,233 @@
+package e2e_test
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"time"
+
+	"github.com/kyma-project/runtime-watcher/skr/tests/e2e/utils"
+	apicorev1 "k8s.io/api/core/v1"
+
+	apiappsv1 "k8s.io/api/apps/v1"
+	apimetav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	"github.com/kyma-project/lifecycle-manager/api/shared"
+	"github.com/kyma-project/lifecycle-manager/api/v1beta2"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+const (
+	controlPlaneNamespace = "kcp-system"
```

Comment:
Some of these constants are redeclared in the `utils/logassert` package. I suggest to use the same constants across the test suite? We need to find a place for a package with ubiquitous constants and reuse as much as possible.